src/test/java/com/dougnoel/sentinel/pages/SampleTestPage.yml

```diff
@@ -0,0 +1,9 @@
+urls:
```

Comment:
If we are going to use this page, it should be in a package named com.encodedna to follow the naming convention of newer pages. This is cleanup that still needs to be done on some of the older page objects.

Separately, I don't know what the permissions are for using this project are. We would be better off leveraging the internet herokuapp domain as it's intended for that use. See if this page will get us what we need: https://the-internet.herokuapp.com/windows

Reply:
Herokuapp doesn't have pop up window, it only has new tab

Reply:
Ok, then add this to a package called com.encodedna in src/test/java